exts/cesium.powertools/cesium/powertools/georefhelper/georef_helper_window.py

```diff
@@ -107,6 +107,32 @@ def set_georef_from_environment():
                 "Cannot set CesiumGeoreference as environment prim does not have latitude or longitude attributes"
             )
 
+    @staticmethod
+    def set_georef_from_anchor():
+        logger = logging.getLogger(__name__)
+        stage = omni.usd.get_context().get_stage()
+
+        cesium_prim = stage.GetPrimAtPath("/CesiumGeoreference")
+
+        if not cesium_prim.IsValid():
+            logger.error("No CesiumGeoreference found")
+            return
+
+        selection = omni.usd.get_context().get_selection().get_selected_prim_paths()
+
+        for prim_path in selection:
+            prim = stage.GetPrimAtPath(prim_path)
+
+            coords = prim.GetAttribute("cesium:anchor:geographicCoordinates").Get()
+            if coords is not None:
+                cesium_prim.GetAttribute("cesium:georeferenceOrigin:latitude").Set(coords[0])
+                cesium_prim.GetAttribute("cesium:georeferenceOrigin:longitude").Set(coords[1])
+                cesium_prim.GetAttribute("cesium:georeferenceOrigin:height").Set(coords[2])
+
```

Comment:
Super-nit, but you've got a blank line before `return` here but not in line 119.